docs/source/index.rst

```diff
@@ -3,41 +3,52 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to xdem's documentation!
+xDEM: Analysis of Digital Elevation Models for geosciences
 ================================
-``xdem`` aims to make Digital Elevation Model (DEM) analysis easy.
-Coregistration, subtraction (and volume measurements), and error statistics should be available to anyone with the correct input data.
+*xdem* aims at simplifying the analysis of Digital Elevation Models (DEM). This module provides typical post-processing steps such as coregistration, elevation/volume change measurements and error statistics.
```

Comment:
DEM with an "s" on this line! :D 
The capitalization is also not so common I think (e.g., https://en.wikipedia.org/wiki/Digital_elevation_model, and I remember it was decapitalized by co-authors several times when I drafted my papers). We should probably decide on one and stay consistent throughout the package. I'd be in favour of lowercase :grin:

Reply:
Used lowercase here. I did not check if this occurred in other areas though.